src/basho_bench_duration.erl

```diff
@@ -126,6 +140,11 @@ terminate(Reason, #state{duration=DurationMins}) ->
     end,
     run_hook(basho_bench_config:get(post_hook, no_op)),
     basho_bench_profiler:maybe_terminate_profiler(basho_bench_config:get(enable_profiler, false)),
+    if Abort =/= true -> ok; true ->
+           ?CONSOLE("Aborting benchmark run...", []),
+           timer:sleep(1000),
```

Comment:
What's the sleep for?

Reply:
To ensure adequate time for the console log message to happen before the halt. Having to trigger `erlang:halt(1)` is a bit of a hack, but I couldn't figure out a better way to promptly kill the VM and also trigger a non zero exit code resulting in the Jenkins job failing.